docs/Makefile

```diff
@@ -5,7 +5,7 @@
 all:  pbench-agent.html
 
```

Comment:
Quick question:  re line 41, is it obvious from context _which_ `requirements.txt` file that is?  (I.e., it's the one in `.../doc/`, and _not_ the one at the base of the Git checkout...which wasn't obvious to _me_...but, perhaps I'm not part of the target demographic....)

(While we're tweaking this file, should we clarify that?)

Reply:
Hmm; should that include a `cd doc`??

Reply:
Makefiles in general, and ours in specific, tend to assume that the files are referenced relative to the directory where the `Makefile` is.  That is, whoever is running this `make` already did the `cd doc` (or, did a `make -C doc`, which is pretty much the same thing).

Now, whether the output of this "build" should be going into our _source_ directory is another question altogether...but that's basically our established practice....  🥴

Reply:
I've updated the reference to be `docs/requirements.txt`.